### ISSUE TITLE
assign all remaining iotempower issues here into the project

### ISSUE THREAD

**Issue by ulno** [iotempire/iotempower#86]

**Comment by ulno** [comment]
estimate [ulno]: 1h

This shoudl be doen soon to not disconnect things

**Comment by mbz4** [comment]
Added workflow automation such that if a PR, or open issue are added, they show up in the Project. If an existing issue is open but not in the project, it won't appear unless the project gets an update (ie, add label, manually add to project).

![Image](https://github.com/iotempire/iotempower/assets/30321314/5b888765-2526-4793-877e-d4a9ba5311b6)

Enabled this workflow.
For new issues, they head to backlog.
For urgent issues have separate item in project already: https://github.com/iotempire/iotempower/issues/97
In short, this one requires custom automation and can be achieved using a new workflow file added to `.github/workflows` directory or repo root directory or others... more in that item. But the idea with urgent is that it should be moved directly to `In Progress` status, to receive priority processing/review/attention/...

These are some thoughts.

**Comment by mbz4** [comment]
@ulno
